### DISCUSSION TITLE
Working time calculation

### DISCUSSION THREAD

**Discussion by ReneEden** [kimai/kimai#5117] [Questions & Answers] [answered]
Hi everyone, 
Is it possible to change the working time calculation during the year? My employees' hours have changed, if I now change the working time calculation, all previous times are also adjusted and my employees have minus hours in the previous months. 

Thank you very much for your support!

Best regards
René Eden

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/documentation/contract.html#changing-a-work-contract

**Reply by ReneEden** [reply]
Many thanks for the quick help and sorry for the trouble, I simply overlooked the point and then didn't look again in the documentation.
